src/app/log-tool/data-setup/import-data/import-data.component.ts

```diff
@@ -175,19 +177,82 @@ export class ImportDataComponent implements OnInit {
     this.dayTypeGraphService.resetData();
     this.invalidFileReferences = new Array();
     if (files[0]) {
-        let extensionPattern: string = '.(json|JSON)$';
+        let extensionPattern: string = '.(json|JSON|gz|GZ)$';
```

Comment:
The log tool part of the implementation can be pulled out since we won't need it anymore